hijack/contrib/admin/templates/hijack/contrib/admin/button.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n hijack %}
+
+{% if request.user|can_hijack:another_user %}
+  <form action="{% url 'hijack:acquire' %}" method="POST">
+    {% csrf_token %}
+    <input type="hidden" name="user_pk" value="{{ another_user.pk }}">
+    {% if is_user_admin %}
+      <button type="submit" class="button">{% trans 'hijack' %}</button>
```

Comment:
Why do you prefer to write text in a button with a small letter? 
It seems to me a "Hijack" or "HIJACK" would look better in a button instead of "hijack".

Reply:
Good point, I will use the `capfirst` filter.

Reply:
I did not know that `trans` can be combined with filters. 
Also I would like to point out that you used the upper filter instead of capfirst.
After this change, the tests fail.

Reply:
Oh yes, thanks. I changed the tests. Upper looked nicer 😬 
And yes, you can. It's very handy if you want to keep natural language in your translation files.